CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ changes.
 
 - Submit observations to a `hydra-explorer` via optional `--explorer` option.
 
+- **BREAKING**
+  - Remove `GetUTxO` client input and corresponding `GetUTxOResponse`. There is already a way to query the `UTxO` in the Head with `GET /snapshot/utxo` query.
+  - Renamed 'CommitFinalized' field 'theDeposit' to 'depositTxId'.
+
```

Comment:
Add this to the changelog: Hydra node API server is used as event sink and should perform better (using bounded memory)

Reply:
Add note about history serving not being the default any more.